src/Microsoft.FileFormats/ELF/ELFStructures.cs

```diff
@@ -138,15 +138,15 @@ public ValidationRule IsProgramHeaderCountReasonable
         {
             get
             {
-                return new ValidationRule("Unreasonably large ELFHeader ProgramHeaderCount", () => ProgramHeaderCount <= 10000);
+                return new ValidationRule("Unreasonably large ELFHeader ProgramHeaderCount", () => ProgramHeaderCount <= 30000);
             }
         }
 
         public ValidationRule IsSectionHeaderCountReasonable
         {
             get
             {
-                return new ValidationRule("Unreasonably large ELFHeader SectionHeaderCount", () => SectionHeaderCount <= 10000);
```

Comment:
What generates so many headers?

Reply:
`createdump` generates them. `createdump` is also going to be changed not to generate all the
"not backed by memory" program header notes which will reduce the number of program headers significantly.